daprdocs/content/en/reference/components-reference/supported-secret-stores/file-secret-store.md

```diff
@@ -56,34 +56,87 @@ Given the following JSON loaded from `secretsFile`:
 }
 ```
 
-If `multiValued` is `"false"`, the store will load the file and create a map with the following key value pairs:
+The flag `multiValued` will decide whether the secret store will present a [name/value behavior or a multiple key-value per secret behavior]({{< ref "secrets_api.md#response-body" >}}).
+
+### Name/Value semantics
+
+
+If `multiValued` is `false`, the store will load [the JSON file]({{< ref "#setup-json-file-to-hold-the-secrets" >}}) and create a map with the following key-value pairs:
 
 | flattened key           | value                           |
 | ---                     | ---                             |
-|"redisPassword"                  | "your redis password"           |
-|"connectionStrings:sql"  | "your sql connection string"    |
-|"connectionStrings:mysql"| "your mysql connection string"  |
+|"redisPassword"          | `"your redis password"`           |
+|"connectionStrings:sql"  | `"your sql connection string"`    |
+|"connectionStrings:mysql"| `"your mysql connection string"`  |
+
 
-Use the flattened key (`connectionStrings:sql`) to access the secret. The following JSON map returned:
+If the `multiValued` setting set to true, invoking a `GET` request on the key `connectionStrings` will result in a 500 HTTP response and an error message. For example:
```

Comment:
Are you sure you mean "true" here? I think you mean "false" for for multiValued here.

Reply:
you are right. Fixing it..

Reply:
And commit fixing this sent. Resolving.